example/spec/discoverGranules/DiscoverGranulesHttpSpec.js

```diff
@@ -81,11 +81,11 @@ describe('The Discover Granules workflow with http Protocol', () => {
         'DiscoverGranules'
       );
     });
-
     it('has expected granules output', () => {
       expect(lambdaOutput.payload.granules.length).toEqual(3);
       expect(lambdaOutput.payload.granules[0].granuleId).toEqual('granule-1');
       expect(lambdaOutput.payload.granules[0].files.length).toEqual(2);
+      expect(lambdaOutput.payload.granules[0].files[0].fileType).toEqual('data');
```

Comment:
Note - the second file (.md5) can have a fileType set, but it will be removed in the protocol download step.   No point in setting/testing it.